pkg/util/virtualmachineinstance/virtualmachineinstance.go

```diff
@@ -104,3 +111,18 @@ func ListByNode(node *corev1.Node, selector labels.Selector, cache ctlkubevirtv1
 	}
 	return list, nil
 }
+
+func VMContainsCDRomOrContainerDisk(vmi *kubevirtv1.VirtualMachineInstance) bool {
```

Comment:
As a public function, suggest to add a first check if vmi is nil, then return false directly.

Reply:
Updated it.